### PR TITLE
[Feature] - Added Badges component in the library

### PR DESCRIPTION
**Pull request by parthpandyappp**
### Directory structure and file justification
Added a new directory named alerts under which markup has been written in `badges.html` and the stylesheet file is written in `badges.css`. The stylesheet file `badges.css` only consists of the styles related to buttons i.e it is a feature-specific stylesheet file that is later getting integrated with the main stylesheet file in the root directory i.e `styles.css`

### Explicit integration of stylesheet file
As we see, many open-source CSS libraries generally have a CDN link that is embedded and linked to the head section of any markup file. Similarly, a single main styles.css file is responsible to provide access to all the other stylesheets.

## About this PR
This PR adds a Badge component of different types.

### CODE REVIEW

File: badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.sub-title {
+  font-size: 1.2rem;
+}
+
+.avatar {
+  border-radius: 50%;
+  height: 100px;
+  width: 100px;
```
Comment (yashpatel1797) [inline badges/badges.css]: if possible maintain consistency in units.

File: badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  border-radius: 50%;
+  border: 1px solid white;
+  background-color: green;
+  height: 20px;
+  width: 20px;
```
Comment (yashpatel1797) [inline badges/badges.css]: you can change units here also.

**Review by yashpatel1797 — CHANGES_REQUESTED** [review]
If possible you can add more examples like different sizes or badges with icons.

File: badges/badges.css (hunk 2 above)
Comment (parthpandyappp) [reply badges/badges.css]: Changed!

File: badges/badges.css (hunk 1 above)
Comment (parthpandyappp) [reply badges/badges.css]: Done, man!

**Review by yashpatel1797 — APPROVED** [review]

File: badges/badges.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
+.disp-badges {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 1rem;
+}
+
+/* 100px = 6.25rem */
+.disp-badge-item {
```
Comment (TejasShekar) [inline badges/badges.css]: Try to use classnames with two words as mentioned by Aneena yesterday.

**Review by TejasShekar — COMMENTED** [review]
Add some more examples. Also, the contrast ratio of the number badge text w.r.t its bg is not good. Have a look into it.

File: badges/badges.css (hunk 3 above)
Comment (parthpandyappp) [reply badges/badges.css]: Okay @TejasShekar . I'll look into it!

**Comment by parthpandyappp** [comment]
Fixed everything!

**Pull request merged**